### ISSUE TITLE
FeedbackRating accessibility

### ISSUE THREAD

**Issue by qroll** [LifeSG/react-design-system#889]
**Component**
`FeedbackRating`

**Describe the feature request**
- Align the FeedbackRating with the accessibility behaviour defined in the Figma specs

**Acceptance criteria**
- Roles should be correctly read out by the AT
- Keyboard support is included
- Minimally test in NVDA + Chrome (Windows), VO + Safari (macOS), Talkback + Chrome (Android)

**References**
[Figma](https://www.figma.com/design/wDAxbRmlq4xOn4UJ8zPuEF/%F0%9F%A7%91%F0%9F%8F%BB%E2%80%8D%F0%9F%A6%AF-Flagship-A11y-Annotations?node-id=1076-4657&p=f&m=dev)
